examples/04_glm_first_level/plot_two_runs_model.py

```diff
@@ -0,0 +1,271 @@
+"""
+Simple example of two-runs fMRI model fitting
+================================================
+
+Here, we will go through a full step-by-step example of fitting a GLM
+to experimental data and visualizing the results.
+This is done on two runs of one subject of the FIAC dataset.
+
+This is a block design experiment with a 2 X 2 experimental design
+with the following factors:
+
+- ``Sentence`` with 2 levels: Same (SSt) vs Different (DSt)
+- ``Speaker``  with 2 levels: Same (SSp) vs Different (DSp)
+
+giving the 4 following conditions:
+
+- Same Sentence-Same Speaker (SStSSp)
+- Same Sentence-Different Speakers (SStDSp)
+- Different Sentences-Same Speaker (DStSSp)
+- Different Sentences-Different Speakers (DStDSp)
+
+The design also included a 5th condition
+containing the first sentence pooled across all conditions.
+
+For more details on the data,
+please see experiment 2 in:footcite:t:`dehaene2006functional`.
+
+Here are the steps we will go through:
+
+1. Set up the GLM
+2. Compare run-specific and fixed effects contrasts
+3. Compute a range of contrasts across both runs
+4. Generate a report
+
+Technically, this example shows how to handle two runs.
+that contain the same experimental conditions.
+The model directly returns a fixed effect
+of the statistics across the two runs.
+"""
+
+# %%
+# Create a write directory to work,
+# it will be a ``results`` subdirectory
+# in the directory of this example file.
+from pathlib import Path
+
+output_dir = (
+    Path(__file__).parent / "results" / Path(__file__).name.replace(".py", "")
+)
+output_dir.mkdir(exist_ok=True, parents=True)
```

Comment:
saves to `examples/04_glm_first_level/results/plot_two_runs_model`

probably should be the way to save the data for all examples

Reply:
well using __file__ does not work